src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/VectorLayerMetadata.java

```diff
@@ -22,7 +10,7 @@
 
 /** Maps to the JSON structure describing a layer of vector tiles */
 @JsonInclude(JsonInclude.Include.NON_NULL)
-class VectorLayerMetadata {
+public class VectorLayerMetadata {
```

Comment:
Does it need to be public?

Reply:
Probably not, in that it worked before. I likely changed it in response to some IDE complaint.

It doesn't seem like something that should be private though, and maybe it might be useful to expose it for additional modules. 

Can revert it (and other visibility changes to `public`) if you'd prefer.

Reply:
I try to keep classes package private unless I have a need for them to be public. Makes long term mainteinance easier to do, less exposed surface that downstream users might be depending onto.
That said, this is a community module, so anyone depending on it does at their own peril.
No need to revert.

Reply:
As it turns out, this is exported through public API e.g. https://github.com/geoserver/geoserver/blob/master/src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TileJSON.java#L53-L57

Knew there was some reason for this...